tests/kas.spec.ts

```diff
@@ -109,12 +108,46 @@ test('test Kafka creation units slider', async ({ page }) => {
   await expect(slider.locator('.pf-c-slider__step-label').last()).toHaveText(config.maxKafkaStreamingUnits.toString());
 });
 
-// TODO: next
-// // test_3kas.py test_kas_kafka_filter_by_status
-// test('test Kafka list filtered by status', async ({ page }) => {
-//   const testInstanceName = `test-instance-${config.sessionID}`;
+const filterByStatus = async function (page, status) {
+  if ((await page.getByRole('button', { name: 'Clear all filters' }).count()) > 0) {
+    await page.getByRole('button', { name: 'Clear all filters' }).click();
+  }
+  await page.getByTestId('large-viewport-toolbar').locator('[aria-label="Options menu"]').click();
+
+  // TODO: Riccardo do you have a better selector here?
+  await page.locator('button[role="option"]:has-text("Status")').click();
```

Comment:
```suggestion
  await page.getByRole("option", {name: "Status"}).click();
```
maybe? It's not exactly equivalent tho.

Reply:
I was using this proposed selector and it wasn't stable enough :-(